packages/broker/src/plugins/metrics/ConsoleAndPM2Metrics.ts

```diff
@@ -94,6 +94,7 @@ export class ConsoleAndPM2Metrics {
     }
 
     stop(): void {
+        io.destroy()
```

Comment:
does this need the typeof function check that petri mentioned?

Reply:
Here the `io.metric` calls in the constructor initialize pm2, so I don't think its necessary.